Allow custom classes for Span attribute (#2753)
strawberry-graphql/strawberry: RELEASE.md

Release type: patch

> This release adds support for custom classes for span to be able to set attributes.

Reviewer: ```suggestion
This release adds support for custom classes inside the OpenTelemetry integration.
```
